src/core/sys/linux/perf_event.d

```diff
@@ -0,0 +1,1873 @@
+/*
+* D header file for perf_event_open system call.
+*
+* Converted/Bodged from linux userspace header
+*
+* Copyright: Max Haughton 2020
+* License: <a href="http://www.boost.org/LICENSE_1_0.txt">Boost License 1.0</a>.
+* Authors: Max Haughton
+ */
+module core.sys.linux.perf_event;
+version (linux):
+extern (C):
+@nogc:
+nothrow:
+@system:
+
+import core.sys.posix.sys.ioctl;
+import core.sys.posix.unistd;
+
+version (X86_64)
+{
+    enum __NR_perf_event_open = 298;
+}
+else version (X86)
+{
+    enum __NR_perf_event_open = 336;
+}
+else version (ARM)
+{
+    enum __NR_perf_event_open = 364;
+}
+else version (ARM64)
+{
+    enum __NR_perf_event_open = 241;
+}
+extern (C) extern long syscall(long __sysno, ...);
```

Comment:
I'm pretty sure we have a binding for this

Reply:
There is not. There are a few PRs but none of them made it in #2820